src/createElements.js

```diff
@@ -37,6 +41,10 @@ export default function createElements(
       return Component ? <Component {...match} /> : null;
     }
 
+    if (route.prerender) {
+      route.prerender({ ...readyState, match });
```

Comment:
i think if we wanted to do that, we'd put `environment` on `match.context`. but i'm gonna try to get `extraQuery` wired up today